src/loader.js

```diff
@@ -30,3 +30,22 @@ export function createLoaderElement() {
   }
   return loader;
 }
+
+
+/**
+ * Creates a default "overflow indicator" element.
+ * @return {!Element}
+ */
+export function createOverflowElement() {
+  let overflow = document.createElement('div');
+  overflow.setAttribute('overflow', '');
+  overflow.classList.add('-amp-overflow');
+  // TODO(dvoytenko): use non-text SVG image to indicate overflow.
+  overflow.textContent = 'more';
```

Comment:
The idea is for a user to specify the whole thing. Like this:
```
<amp-iframe src="...">
  <div overflow aria-lable="My text">MY TEXT <svg/></div>
</amp-iframe>
```
This is similar to loading indicator and other cases. The thinking here is that it's easier to do the wholesome replacement that do attribute-by-attribute.